problems/getCountriesSortedByPopulation.js

```diff
@@ -7,8 +7,14 @@
 * 
 */
 
-function getCountriesSortedByPopulation() {
-
+function getCountriesSortedByPopulation(arr) {
+  arr.sort((country1, country2) => {
+    return country2.population - country1.population;
+  });
+  return arr.map((country) => {
```

Comment:
This is great! 

Another way to tackle this is to just add map on to your sort!